axlearn/common/trainer.py

```diff
@@ -964,8 +973,6 @@ def _get_compiled_train_step_fn(
     ) -> Callable[[TrainerState, NestedTensor], tuple[TrainerState, NestedTensor]]:
         """Build a fully compiled train step function.
 
-        Relies on the JAX pjit cache to avoid recompilation where possible.
```

Comment:
Relies on the JAX pjit cache to avoid recompilation when with_xsc=True or enable_python_train_step_cache=False.

Reply:
Done.